cinder/volume/targets/rdma.py

```diff
@@ -0,0 +1,54 @@
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+from oslo_log import log as logging
+
+from cinder.volume.targets import driver
+
+LOG = logging.getLogger(__name__)
+
+
+class RDMATarget(driver.Target):
+    """Target object for block storage devices with RDMA transport.
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(RDMATarget, self).__init__(*args, **kwargs)
+        self.protocol = 'RDMA'
```

Comment:
So, we need some consistency everywhere https://github.com/openstack/cinder/blob/master/cinder/volume/targets/iscsi.py#L42